.github/workflows/v2-build-and-deploy.yml

```
@@ -0,0 +1,97 @@
+name: V2 Build and Deploy Demos
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: SWC environment to deploy to
+        options:
+          - swc-staging
+          - swc-prod
+          - swc-dev
+        required: true
+        type: choice
+      target:
+        default: stable
+        description: PennyLane version to build the demos. Either 'latest' or the most recent 'stable' release.
+        options:
+          - latest
+          - stable
+        required: true
```

This shouldn't be required if there's a default value.
```suggestion
        required: false
```